payments/fixtures/berth_products.json

```diff
@@ -1,4 +1,20 @@
 [
+{
+    "model": "payments.berthproduct",
+    "pk": "0fa2a826-771b-4682-833f-1d7d20fa840d",
+    "fields": {
+        "created_at": "2021-05-10T12:05:25.521Z",
+        "modified_at": "2021-05-10T12:11:25.266Z",
+        "tax_percentage": "24.00",
+        "min_width": "2.99",
```

Comment:
I don't think they will change anytime soon. In case they did, it would be pretty straightforward to update the fixture and the products in production to fit the case.